ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/wizard/v1/wizard/wizard.html

```diff
@@ -54,11 +56,10 @@
             </ol>
         </div>
 
-        <div
-                class="cmp-adaptiveform-wizard__previousNav"
-                tabindex = 0
-                role="navigation"
-                aria-label="Previous Button">
+        <div class="cmp-adaptiveform-wizard__button-container">
+          <div class="cmp-adaptiveform-wizard__previousNav" tabindex = 0 role="navigation" aria-label="Previous Button"></div>
+          <div class="cmp-adaptiveform-wizard__nextNav" tabindex = 0 role="navigation" aria-label="Next Button">
```

Comment:
Ideally for nav, this should be the BEM to signify the modifier class,

 <div class="cmp-adaptiveform-wizard__nav cmp-adaptiveform-wizard__nav--previous" tabindex="0" role="navigation" aria-label="Previous Button"></div>
    <div class="cmp-adaptiveform-wizard__nav cmp-adaptiveform-wizard__nav--next" tabindex="0" role="navigation" aria-label="Next Button"></div>

Reply:
changed

Reply:
I don't see the changes, please fix this.

Reply:
this is old class. I just changed this position.